calico-enterprise/operations/clis/calicoctl/install.mdx

```diff
@@ -122,8 +122,8 @@ to a location that's in your `PATH`. For example, `C:\Windows`.
 
 <CodeBlock language='bash'>
   {'{{version}}' === 'master'
-    ? `Invoke-WebRequest -Uri "{{filesUrl}}/download/binaries/${version}/calicoctl-windows-amd64.exe" -OutFile "calicocttl.exe"`
-    : `Invoke-WebRequest -Uri "{{downloadsurl}}/ee/binaries/${version}/calicoctl-windows-amd64.exe" -OutFile "calicocttl.exe"`}
+    ? `Invoke-WebRequest -Uri "{{filesUrl}}/download/binaries/${version}/calicoctl-windows-amd64.exe" -OutFile "calicoctl.exe"`
```

Comment:
@coutinhop Can you confirm this fix is good?

Reply:
@ctauchen yes, it's good! thanks @jsoref for catching these